Nix: update and add documentation for nix subcommands (#9959)
tldr-pages/tldr: pages/common/nix-classic.md

# Nix classic interface

> A classic, stable interface to a powerful package manager that makes package management reliable, reproducible, and declarative.

> > For an improved, experimental interface, see `tldr nix`.
> > Some Nix commands such as `nix-build`, `nix-shell`, `nix-env`, and `nix-store` have their own pages.

Reviewer: ```suggestion
> Some Nix commands such as `nix-build`, `nix-shell`, `nix-env`, and `nix-store` have their own pages. See also: `tldr nix`.
```
